guides/doc-Content_Management_Guide/topics/Containers_interacting_with_registry.adoc

```diff
@@ -0,0 +1,32 @@
+[[Using_Container_Registries]]
+= Using Katello Container Registries
+
+Interacting with Katello container registries should provide nearly the same experience as using any other container registry.
+
+Pushing container images is the most notable feature that is not supported by Katello.
+
+.Container Registries on Smart Proxies
+
+On Smart Proxies with content, the https://github.com/Katello/smart_proxy_container_gateway[Container Gateway] smart proxy plugin acts as the container registry.  It caches authentication information from Katello and proxies incoming requests to Pulp.  The Container Gateway comes default on Smart Proxies with content.
+
+*Please note* that, at first release with Katello 4.0, the Container Gateway will not yet support pulling container images that require authentication.  Until it does, please ensure that *Unauthenticated Pull* is checked for all Lifecycle Enviroments that contain container repositories that are expected to be served by Smart Proxies.  This is planned to be resolved by Katello 4.1.
```

Comment:
I think we should introduce Limitations header like we often do in other cases. That's usually better way to document limitations.